website/docs/components/code-block/index.md

```diff
@@ -28,4 +28,4 @@ navigation:
 
 <section data-tab="Accessibility">
   @include "partials/accessibility/accessibility.md"
-</section>
\ No newline at end of file
+</section>
```

Comment:
Whoops, I made this change it GitHub’s editor, so I guess it either added or deleted a `\n`. Not sure which!

Reply:
Yeah, we're not so much in sync with GitHub's default new line rules (as we expect then not to exist). Fixed it now